### ISSUE TITLE
Language independent? (Or adding German)

### ISSUE THREAD

**Issue by phlind** [tadashi-aikawa/obsidian-various-complements-plugin#3]
Would it be possible to make this language independent? For example, I assume that in Sublime text the auto complete probably works like that. If not, it would be great if German could be added.

**Comment by tadashi-aikawa** [comment]
Hi @phlind!

It might be possible to support German by configuring languages. I have checked how this plugin tokenizes German. It doesn't seem to be what I expected.

```
Mit diesem Plugin für Obsidian können Sie Eingaben in Markdown-Dateien ergänzen.
↓
Mit | | diesem | | Plugin | | f | ü | r | | Obsidian | | k | ö | nnen | | Sie | | Eingaben | | in | | Markdown | - | Dateien | | erg | ä | nzen | .
```

Whereas, using wink-tokenizer, which supports German, looks good as follows.

```
[
  { value: 'Mit', tag: 'word' },
  { value: 'diesem', tag: 'word' },
  { value: 'Plugin', tag: 'word' },
  { value: 'für', tag: 'word' },
  { value: 'Obsidian', tag: 'word' },
  { value: 'können', tag: 'word' },
  { value: 'Sie', tag: 'word' },
  { value: 'Eingaben', tag: 'word' },
  { value: 'in', tag: 'word' },
  { value: 'Markdown', tag: 'word' },
  { value: '-', tag: 'punctuation' },
  { value: 'Dateien', tag: 'word' },
  { value: 'ergänzen', tag: 'word' },
  { value: '.', tag: 'punctuation' }
]
```

If the above is what you expected, I want to implement it. What do you think?

**Comment by phlind** [comment]
> Whereas, using [wink-tokenizer](https://www.npmjs.com/package/wink-tokenizer), which supports German, looks good as follows.
> 
> ```
> [
>   { value: 'Mit', tag: 'word' },
>   { value: 'diesem', tag: 'word' },
>   { value: 'Plugin', tag: 'word' },
>   { value: 'für', tag: 'word' },
>   { value: 'Obsidian', tag: 'word' },
>   { value: 'können', tag: 'word' },
>   { value: 'Sie', tag: 'word' },
>   { value: 'Eingaben', tag: 'word' },
>   { value: 'in', tag: 'word' },
>   { value: 'Markdown', tag: 'word' },
>   { value: '-', tag: 'punctuation' },
>   { value: 'Dateien', tag: 'word' },
>   { value: 'ergänzen', tag: 'word' },
>   { value: '.', tag: 'punctuation' }
> ]
> ```
> 
> If the above is what you expected, I want to implement it. What do you think?

Yes, this looks good!

**Comment by tadashi-aikawa** [comment]
Hi @phlind.

I have released v0.3.0, and it supports languages whose word break is whitespace (maybe German is too😉). Could you confirm it?

**Comment by phlind** [comment]
@tadashi-aikawa Yes, I just tested it and it seems to work fine. Thanks a lot, that's a great convenience option. :)

**Comment by tadashi-aikawa** [comment]
Thank you for testing!  I will close this issue.

**Comment by tadashi-aikawa** [comment]
Fixed `v0.3.0`
